tests/conformance/retry_strategy_test_data.json

```diff
@@ -85,13 +85,28 @@
       },
       {
         "id": 4,
-        "description": "non idempotent",
+        "description": "non_idempotent",
         "cases": [
           {
-            "instructions": []
+            "instructions": ["return-503"]
```

Comment:
We should make a separate PR at some point with additional instruction sets-- also, I see this doesn't exactly match what you have in your conformance-tests repo PR.

Reply:
Well noted! The difference now is the # of instructions cases.

Reply:
Added instruction cases. S1 ~ S4 now fully matches the conformance-tests repo json.